app/views/restaurants/show.html.haml

```diff
@@ -9,22 +9,35 @@
 %p= @restaurant.address
 %p= @restaurant.description
 - @dishes.each do |dish|
-  %p
-    Dish:
-    = dish.name
-  %p
-    Description:
-    = dish.description
-  %p
-    Category:
-    = dish.category
-  %p
-    Restaurant:
-    = dish.restaurant
-  %p
-    Price:
-    = dish.price
-    $
-  %img{alt: dish.name, src: dish.pic_url}/
+  %div{id: dish.name}
+    %p
+      Dish:
+      = dish.name
+    %p
+      Description:
+      = dish.description
+    %p
+      Category:
+      = dish.category
+    %p
+      Restaurant:
+      = dish.restaurant
+    %p
+      Price:
+      = dish.price
+      $
+    %img{alt: dish.name, src: dish.pic_url}/
+    %form
```

Comment:
This form needs to point to a route. Use form helpers

Reply:
Yeah tried a lot of different things regarding form helpers, Gustaf is helping me now :)